src/app/onion/dashboard/components/splash/splash.component.ts

```diff
@@ -1,9 +1,25 @@
 import { Component, EventEmitter, Output } from '@angular/core';
+import { trigger, transition, style, animate, animateChild, query } from '@angular/animations';
 
 @Component({
   selector: 'clark-dashboard-splash',
   templateUrl: './splash.component.html',
-  styleUrls: ['./splash.component.scss']
+  styleUrls: ['./splash.component.scss'],
+  animations: [
+    trigger('splash', [
+      transition(':enter', [
+        style({width: '0px', 'padding-left': '0px', 'padding-right': '0px', opacity: 0}),
```

Comment:
Can we convert these animations to use `transform: scale(0, 1)` to `transform: scale(1)` instead of the width/padding hybrid? I know these are carbon-copied from the old dashboard but I did them wrong.
![image](https://user-images.githubusercontent.com/7431390/62242168-8fcd8800-b3a8-11e9-86cf-d3c849641316.png)